shared-cluster/create-cluster

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# This script runs in pod in namespace 'bugzilla-operator'
+# oc is installed in PATH
+# /cache persistent volume mounted
+
+# To run locally: 
+# $ podman run --rm -it quay.io/sallyom/groupb-shared-cluster:4.8 \
+#      -v /path/to/artifacts:${CLUSTER_DIR}:z \
+#      -v /path/to/pullsecret:${PS_PATH}:z \
+#      --env-host=true \
+#      create-cluster
+
+# set in controller w/ os.Setenv from operatorConfig.SharedCluster
+# export AWS_PROFILE="${AWS_PROFILE}"
+# export AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}"
+# export AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID}"
+# NAME=groupbMMDDHH
+# export CLUSTER_NAME="${NAME}"
+# export CLUSTER_DIR="/cache/groupbclusters/${NAME}"
+# export PS_PATH=/path/pullsecret
+# export BASE_DOMAIN=group-b.devcluster.openshift.com
+# export PUB_SSH_KEY=pubKeyString
+
+# This needs an update every release
+# export LATEST_CI=4.9.0-0.ci
+
```

Comment:
@sanchezl note the env vars needed, see operatorConfig.SharedCluster 
(might use a configmap or source an env file instead, see: https://github.com/mfojtik/bugzilla-operator/pull/142/files#r684636984

Reply:
see the rest of that file for the exported vars